common/services/apm/apmConfig.js

```diff
@@ -1,8 +1,16 @@
-module.exports = serviceName => ({
+const commonApmConfig = serviceName => ({
   serviceName,
   environment: process.env.APM_ENVIRONMENT,
   serverUrl: process.env.APM_SERVER_URL,
   // we've used inactive so by default it is on, and we can purposefully turn it off locally
   active: process.env.APM_INACTIVE !== 'true',
   centralConfig: true,
 });
+
+const client = commonApmConfig;
+const server = serviceName => ({
+  ...commonApmConfig(serviceName),
+  secretToken: process.env.APM_SECRET,
```

Comment:
Do we not need to add this to the terraform of the apps then to exist? Assuming it just exists on your local machine?

Reply:
Ha, yes, knew I was missing something

Reply:
Turned out we'd never added the config for RUM to the identity app TF either!

Reply:
I really like the way vercel uses a `vercel env` command to generate values locally from remote... something to look into.

https://vercel.com/docs/concepts/projects/environment-variables